repos/system_upgrade/el7toel8/libraries/config/mock_configs.py

```diff
@@ -0,0 +1,33 @@
+from leapp.models import IPUConfig, EnvVar, OSRelease, Version
+
```

Comment:
Maybe little too late, but couldn't be better to put a discaimer into the file? I knot that there is *mock_* prefix in the name, but to ensure that people will not use it in the standard code

Reply:
I'm about to merge #340 in a while.